### ISSUE TITLE
When download a remote file,How to overwrite local file?

### ISSUE THREAD

**Issue by 18718821950** [liximomo/vscode-sftp#671]
**Operation recurrence:**

1. Modify a file randomly in the workspace without saving or uploading.

2. Right click this file to download

**Result:**

There is no change in the content of the document. The content just modified is still in use.

**Expect:**

The contents of the remote file overwrite the local

**Comment by cooper1x** [comment]
me too

**Comment by zetokore** [comment]
I agree, the default behaviour is broken.  It should overwrite your local file on download.  Otherwise, what exactly is the download command doing?
